.github/workflows/release.yml

```diff
@@ -11,10 +11,10 @@ jobs:
     steps:
       -
         name: Checkout
-        uses: actions/checkout@v2
-      -
-        name: Unshallow
-        run: git fetch --prune --unshallow
+        uses: actions/checkout@v4
+        with:
```

Comment:
Why do we need this here and not above?

Reply:
The release action needs to know the current tag. This is not populated by default and needs the unshallow.